config/secrets.yaml.template

```diff
@@ -0,0 +1,14 @@
+batch:
```

Comment:
can we add a comment at the top of this file that links back to the docs to show people who to create their batch/storage accounts